tomviz/python/Recon_TV_minimization.py

```diff
@@ -4,9 +4,9 @@
 import time
 
 
-class ReconTVOperator(tomviz.operators.CancelableOperator):
+class ReconTVOperator(tomviz.operators.CompletableOperator):
 
-    def transform(self, dataset, Niter=10, Nupdates=0):
+    def transform(self, dataset, Niter=10, Nupdates=0): # noqa
```

Comment:
Please specify the flake8 error code that you are trying to suppress to make things clear to the reader and to ensure we do not unintentionally suppress other error codes.
```suggestion
    def transform(self, dataset, Niter=10, Nupdates=0): # noqa: X123
```